### DISCUSSION TITLE
一部のキー（SとZ）が反応しません。

### DISCUSSION THREAD

**Discussion by 1280560** [Yowkees/keyball#712] [Q&A]
こんにちは！初めての自作キーボード楽しく組み立てています。
ただ、Sキー、Ｚキーが反応せず困っていまして、原因特定していただけると嬉しいです。いくつかハンダのつけ直しもしましたが、改善しませんでした。よろしくお願いします…！
![IMG_4271](https://github.com/user-attachments/assets/0c07be37-99da-4185-8624-a74e49bfdad6)

**Comment by 1280560** [comment]
【解決しました】
キーソケットのハンダを温め直したり、ハンダを追加したりしていたら、無事に動作しました！
お騒がせしました！

**Comment by Yowkees** [comment]
ご自身で解決していただき、誠にありがとうございます。
PCBソケットはとても半田付けが難しくて恐縮です。私もいまだにテンプラしてしまうことがあります。
ぜひ末永くKeyball39をご愛護いただければ幸いです。
